packages/codegen/src/index.test.ts

```diff
@@ -60,23 +60,28 @@ describe("generateSource", () => {
     expect(src).toContain(`export type Option = ("one" | "two" | "three")[];`);
   });
 
-  it("should handle properties both inside and outside of allOf", async () => {
+  it(async () => {
     const src = await generate(__dirname + "/__fixtures__/allOf.json");
-    expect(src).toContain(
-      "export type Circle = Shape & { radius?: number; } & { circumference?: number; };",
-    );
-  });
 
-  it("should support discriminator used in conjunction with allOf", async () => {
-    const src = await generate(__dirname + "/__fixtures__/allOf.json");
-    expect(src).toContain("export type PetBase = { petType: string; };");
-    expect(src).toContain("export type Pet = Dog | Cat | Lizard;");
-    expect(src).toContain(
-      'export type Dog = { petType: "dog"; } & PetBase & { bark?: string; };',
-    );
-    expect(src).toContain(
-      'export type Lizard = { petType: "Lizard"; } & PetBase & { lovesRocks?: boolean; };',
-    );
+    it("should handle properties both inside and outside of allOf", async () => {
+      expect(src).toContain(
+        "export type Circle = Shape & { radius?: number; } & { circumference?: number; };",
+      );
+    });
+
+    it("should support discriminator used in conjunction with allOf", async () => {
+      expect(src).toContain("export type PetBase = { petType: string; };");
+      expect(src).toContain(
+        'export type Cat = { petType: "cat"; } & PetBase & { name?: string; };',
+      );
+      expect(src).toContain(
+        'export type Dog = { petType: "dog" | "poodle"; } & PetBase & { bark?: string; };',
+      );
+      expect(src).toContain(
+        'export type Lizard = { petType: "Lizard"; } & PetBase & { lovesRocks?: boolean; };',
+      );
+      expect(src).toContain("export type Pet = Dog | Cat | Lizard;");
```

Comment:
Deduplicated members here now.

Reply:
I read this as "Prior to the fix, there were duplication here". But we had the same assertion in line 73 before. 

So how is this test validating the fix, or something else?

Reply:
...assuming that the duplication was added along with the fix, right? So Current version doesn't have duplication but with the change from `find` to `filter` it was added, then mitigated using `_.uniq` so that this assertion remains valid, right?